packages/mjml-core/src/helpers/traverseMJML.js

```diff
@@ -3,9 +3,13 @@ import head from 'lodash/head'
 import map from 'lodash/map'
 import noop from 'lodash/noop'
 
-export default function traverseMJML(mjml, predicate = noop) {
+export default function traverseMJML(mjml, predicate = noop, ignoreTagContent = null) {
```

Comment:
Would prefer to have an `options = {}` here instead of a named string.
String parameters can be hard to read when you don't have any context